### PR TITLE
CSS two column layout.

### PR DESCRIPTION
**Pull request by louiewhitz**
Screen Shot 2022-05-14 at 9 52 42 PM

### CODE REVIEW

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Louisa, your output doesn't match the example output in multiple ways. For instance, your fonts are incorrect in multiple places, and your content is not vertically aligned in the right set of columns. Please revise and submit a new screenshot with your revision. 

image

**Comment by louiewhitz** [comment]
Screen Shot 2022-05-22 at 6 05 50 PM

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Louisa, this isn't making use of proper layout classes. I see you have a `container` class, but you don't have a `row` or any `column` classes. Please revise this to make use of proper layout classes.

**Review by louiewhitz — COMMENTED** [review]
Screen Shot 2022-05-27 at 8 05 25 PM

File: css-two-column-layout/index.html [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  <div class="container">
+    <div class="py-7 text-center">
+      <h1>Scale, Speed, Agility</h1>
+      <p class="paraheader">We've dedicated thousands of man hours to making healthcare institutions
+        in the Helium network run 10x more efficiently and deliver an unparralled quality of care.
+      </p>
+    </div>
+    <div class="info-card-grid">
+      <div class="row">
+        <div class="info-card-image-column">
```
Comment (codingcodymiller) [inline css-two-column-layout/index.html]: `column` classes should be named after their size, not their content. Make sure you address this on any other `column` classes in this document as well.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**